ls1intum/Artemis
# Bamboo: Set Build Plan Premission after copy

### Checklist

### Motivation and Context
#1567

### Description
Update Permissions when importing programming exercises from other courses.

### Steps for Testing


1. Import programming exercise from other course with different user groups for instructor / ta
2. Check Bamboo that plan premissions match Artemis.
